net/Socket.cpp

```diff
@@ -555,6 +555,33 @@ void StreamSocket::dumpState(std::ostream& os)
         Util::dumpHex(os, "\t\toutBuffer:\n", "\t\t", _outBuffer);
 }
 
+void StreamSocket::sendError(int errorCode,
+               const std::string& body, const std::string& extraHeader, bool shutdownAfter)
+{
+
+    std::ostringstream oss;
+    oss << "HTTP/1.1 " << errorCode << "\r\n"
+        << "Date: " << Util::getHttpTimeNow() << "\r\n"
+        << "User-Agent: " << WOPI_AGENT_STRING << "\r\n"
+        << extraHeader;
+    if (body.empty())
+    {
+        oss << "Content-Length: 0\r\n";
```

Comment:
Why not just unconditionally set the content-length to the body size?

`oss << "Content-Length: " << body.size() << "\r\n";`

Reply:
Yep, changed in d398545e5